### PR TITLE
create readme

### PR DESCRIPTION
**Pull request by aeddaqqa**
create readme to describe how to start the app with wallet and explorer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+1. Suite App, built with React
+2. Camino explorer, built with React
+3. Camino Wallet, built with Vue
+
+## Prerequisites
+
+- Node v16 or higher
+- Camino explorer runing
+- Camino wallet running
```
Comment (Chriserus) [inline README.md]: Running*

File: README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Camino Application Suite
+
+This app uses Module Federation to combine three different applications with different frameworks into a single, cohesive application. The three applications are:
+
+1. Suite App, built with React
+2. Camino explorer, built with React
```
Comment (Chriserus) [inline README.md]: "Explorer"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## Prerequisites
+
+- Node v16 or higher
+- Camino explorer runing
+- Camino wallet running
+
+## Install the dependencies
+
+To get started you need to have explorer and wallet running, first clone repositories
```
Comment (Chriserus) [inline README.md]: Remove ", first clone repositories", as it's the first step anyway

File: README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    4. start the development server for the application
+
+        ```bash
+        yarn start
+        ```
+
+## Getting Started
+
+after succefly run the explorer and wallet, clone this repository and navigate to the root directory in your terminal.
```
Comment (Chriserus) [inline README.md]: "After successfully running the explorer..."

File: README.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+    4. start the development server for the application
+
+        ```bash
+        yarn start
+        ```
+
+## Getting Started
+
+aAfter successfully running the explorer and wallet, clone this repository and navigate to the root directory in your terminal.
```
Comment (Chriserus) [inline README.md]: "aAfter"?

**Review by Chriserus — APPROVED** [review]

**Review by Ayagoumi — APPROVED** [review]

**Pull request merged**